add expo-dev-client transactions fix to troubleshooting (#6071)
getsentry/sentry-docs: src/platforms/react-native/troubleshooting.mdx

Note that if the source maps upload fails, you'll have to do it [manually](/platforms/react-native/sourcemaps) later when the connection is restored.

When you remove the environmental variable in front of your build command, your build will work as before.

> ## Expo transactions never finish
> 
> If you're using [expo-dev-client](https://docs.expo.dev/development/introduction/#what-is-an-expo-dev-client), you might notice that transactions never finish in your dev builds. This is due to logs that the dev client is sending to the development server. To fix this behavior in your dev builds, you can stop creating spans for the HTTP requests to the log endpoint. To do this, add the following to your `Sentry.init()`:
> 

Reviewer: ```suggestion
## Expo Transactions Never Finish

If you're using [expo-dev-client](https://docs.expo.dev/development/introduction/#what-is-an-expo-dev-client), you might notice that transactions never finish in your dev builds. This is due to logs that the dev client is continuously sending to the development server. To fix this, you can stop creating spans for the HTTP requests to the log endpoint in your dev builds by adding the following to your `Sentry.init()`:

```
